hack/make/tests/e2e.mk

```diff
@@ -26,7 +26,7 @@ test/e2e-publish:
 
```

Comment:
Although code LGTM, after running the standard scenario several times I still got some errors. I am not sure whether this is only me and my local set up, so please if anyone can confirm that they are also getting errors, reply here.

Example (this time it is the withoutcsi test):
```
    app.go:151: 
                Error Trace:    /Users/albian.krasniqi/Library/CloudStorage/OneDrive-Dynatrace/Desktop/Proj/dynatrace-operator/test/helpers/sample/app.go:151
                                                        /Users/albian.krasniqi/go/pkg/mod/sigs.k8s.io/e2e-framework@v0.4.0/pkg/env/env.go:444
                                                        /Users/albian.krasniqi/go/pkg/mod/sigs.k8s.io/e2e-framework@v0.4.0/pkg/env/env.go:483
                Error:          Received unexpected error:
                                client rate limiter Wait returned an error: context deadline exceeded
                Test:           TestStandard/app-monitoring-without-csi/install_already_injected_sample_app
=== NAME  TestStandard/app-monitoring-without-csi
    app.go:178: 
                Error Trace:    /Users/albian.krasniqi/Library/CloudStorage/OneDrive-Dynatrace/Desktop/Proj/dynatrace-operator/test/helpers/sample/app.go:178
                                                        /Users/albian.krasniqi/go/pkg/mod/sigs.k8s.io/e2e-framework@v0.4.0/pkg/env/env.go:444
                                                        /Users/albian.krasniqi/go/pkg/mod/sigs.k8s.io/e2e-framework@v0.4.0/pkg/env/env.go:507
                Error:          Received unexpected error:
                                deployments.apps "random-user" not found
                Test:           TestStandard/app-monitoring-without-csi
--- FAIL: TestStandard (424.41s)
```